recipes/scikit-gstat/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "scikit-gstat" %}
+{% set version = "0.5.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3b0c423a841be6f23ce99f3f1365b64044d57a4a64d58e36ad2b773ea5805300
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+
+  run:
+    - imageio
+    - matplotlib
```

Comment:
Unless you actually want the `pyqt` GUI, better to depend on `matplotlib-base`.

Reply:
Thanks, @dopplershift! Changed in efc134d